### ISSUE TITLE
Segmentation Issue [2]

### ISSUE THREAD

**Issue by VirusMS** [samp-incognito/samp-streamer-plugin#297]
#296 

@samp-incognito I disagree.

'''
public OnPlayerConnect(playerid)
{
  return 1;
}
'''

Still causes Segmentation Fault. Now, I am afraid, it can be claimed as OnPlayerConnect issue, since it's obviously a broken hook of sorts, and it is rather obvious.

Still need your help ASAP.

**Comment by samp-incognito** [comment]
The first line in the backtrace references some code that's hooking OnPlayerConnect in one of your scripts. The streamer plugin doesn't use PAWN-based callback hooks, so it can't be responsible. Additionally, if the streamer plugin did actually cause a crash, it would normally be evident from the backtrace (i.e., streamer.so would appear somewhere near the top).

You must have some code that's causing OnPlayerConnect to crash the server, whether it's in an include file or a filterscript. Again, as I suggested, try commenting out portions of your code until it no longer crashes.

**Comment by VirusMS** [comment]
@samp-incognito 
I do not mean to offend, but I have to point out for you that I only used return 1 in OnPlayerConnect callback, which means there was no code in it. At all. Which makes it a hook to be responsible, because there is basically no code.

**Comment by samp-incognito** [comment]
> I do not mean to offend, but I have to point out for you that I only used return 1 in OnPlayerConnect callback, which means there was no code in it. At all. Which makes it a hook to be responsible, because there is basically no other reason you could state in such condition.

Right, and I've already said that the streamer plugin doesn't use script-based callback hooks. Something in one of your scripts (maybe an include file) is hooking a callback and doing this.

Have you even tried running without the streamer plugin loaded to see if the server is still crashing?

**Comment by VirusMS** [comment]
Amount of changes I have to make in the code is big enough for me to rather expect you to help me with it instead of spending my time on changing all these lines myself, since it'd be much faster.

I believe you, but I basically thought you could provide help. If you may not, or in case you refuse to, close the thread and I will not bugger you again. Thanks.

**Comment by VirusMS** [comment]
I've tested all the includes one by one in an empty gamemode and found the solution. THANK YOU, I SPENT A WEEK TRYING TO FIGURE!

**Comment by VinDieseLL** [comment]
hwo you solve this?

**Comment by BedulInHare** [comment]
how do you solve the problem sir?
